menus/julia-client.cson

```diff
@@ -9,6 +9,7 @@
         {label: 'Select Block', command: 'julia-client:select-block'}
         {label: 'Go to Definition', command: 'julia-client:goto-symbol'}
         {label: 'Show Documentation', command: 'julia-client:show-documentation'}
+        {label: 'Format Code', command: 'julia-client:format-code'}
```

Comment:
Maybe also add an entry to `lib/package/menu.coffee`?

Reply:
Or even a toolbar button. This feature is something lots of people have asked for, so I feel it's warranted to give it some exposure.